administration/troubleshooting.md

```diff
@@ -277,108 +288,102 @@ Fluent Bit v2.0.8
-We chose a filter record since it includes the most details of all the record types.
+This filter record is an example to explain the details of a Tap record:
 
 ```json
 {
-	"type": 2,
-	"start_time": 1666349231,
-	"end_time": 1666349231,
-	"trace_id": "trace.1",
-	"plugin_instance": "nest.2", 
-	"records": [{
-		"timestamp": 1666349231,
-		"record": {
-			"dummy": "dummy_0",
-			"powered_by": "fluent",
-			"data": {
-				"key_name": "foo", 
-				"key_cnt": "1"
-			}
-		}
-	}]
+   "type": 2,
+   "start_time": 1666349231,
+   "end_time": 1666349231,
+   "trace_id": "trace.1",
+   "plugin_instance": "nest.2",
+   "records": [{
+      "timestamp": 1666349231,
+      "record": {
+         "dummy": "dummy_0",
+         "powered_by": "fluent",
+         "data": {
+            "key_name": "foo",
+            "key_cnt": "1"
+         }
+      }
+   }]
 }
 ```
 
-### type
-
-The type defines at what stage the event is generated:
-
-- type=1: input record
-  - this is the unadulterated input record
-- type=2: filtered record
-  - this is a record once it has been filtered. One record is generated per filter.
-- type=3: pre-output record
-  - this is the record right before it is sent for output.
-
-Since this is a record generated by the manipulation of a record by a filter is has the type `2`.
-
-### start_time and end_time
-
-This records the start and end of an event, it is a bit different for each event type:
-
-- type 1: when the input is received, both the start and end time.
-- type 2: the time when filtering is matched until it has finished processing.
-- type 3: the time when the input is received and when it is finally slated for output.
-
-### trace_id
-
-This is a string composed of a prefix and a number which is incremented with each record received by the input during the Tap session.
-
-### plugin_instance
-
-This is the plugin instance name as it is generated by Fluent Bit at runtime.
-
-### plugin_alias
-
-If an alias is set this field will contain the alias set for a plugin.
-
-### records
-
-This is an array of all the records being sent. Since Fluent Bit handles records in chunks of multiple records and chunks are indivisible the same is done in the Tap output. Each record consists of its timestamp followed by the actual data which is a composite type of keys and values.
+- `type`: Defines the stage the event is generated:
+  - `1`: Input record. This is the unadulterated input record.
+  - `2`: Filtered record. This is a record once it has been filtered. One record is
+    generated per filter.
+  - `3`: Pre-output record. This is the record right before it's sent for output.
+
+  This example is a record generated by the manipulation of a record by a filter so
+  it has the type `2`.
+- `start_time` and `end_time`: Records the start and end of an event, and is
+  different for each event type:
+  - type 1: When the input is received, both the start and end time.
```

Comment:
Should `type 1` and the others be in backticks? Not sure if they're actual values.

Reply:
iffy, if it was just `1`, yes, but here I decided to leave it.

I think that's a thing that may need overall reviewing.